malevo/swagger.yml

```diff
@@ -0,0 +1,174 @@
+swagger: '2.0'
+info:
+  description: Malevo API
+  version: 0.1.0
+  title: Malevo API
+#host: localhost
+basePath: /api/malevo
+tags:
+  - name: class
+    description: Everything about predicted and ground truth classes
+  - name: images
+    description: Everything about images
+schemes:
+  - http
+paths:
+  /class/sampleImages:
+    get:
+      tags:
+        - class
+      summary: Representative sample of images
+      description: >-
+        Returns a list of representative images for that matches all given class
+        ids
+      produces:
+        - application/json
+      parameters:
+        - name: dataset
+          in: query
+          description: Phovea dataset id
+          required: true
+          type: string
+        - name: epoch
+          in: query
+          description: Selected epoch id
+          required: true
+          type: integer
+        - name: groundTruthClassId
```

Comment:
do we need the `...ClassId` suffix in the call? Either add `...Id` to all names (e.g. epoch, dataset/run) or none